src/app/ui/components/tooltip/basic-tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+
+import * as RadixTooltip from '@radix-ui/react-tooltip';
+
+import { Tooltip } from './tooltip';
+
+export function BasicTooltip({
+  children,
+  label,
+  disabled,
+  side,
+}: {
+  children: ReactNode;
+  label?: string;
+  disabled?: boolean;
+  side?: RadixTooltip.TooltipContentProps['side'];
+}) {
```

Comment:
```suggestion
interface BasicTooltipProps {
  children: ReactNode;
  label?: string;
  disabled?: boolean;
  side?: RadixTooltip.TooltipContentProps['side'];
}
export function BasicTooltip({
  children,
  label,
  disabled,
  side,
}: BasicTooltipProps) {
```
extract to separate prop